### PR TITLE
Add GOV.UK lint gem

### PR DESCRIPTION
**Pull request by himal-mandalia**
Adds [GOV.UK lint](https://github.com/alphagov/govuk-lint) and CI configuration for Azure DevOps.

Note: There will be a number of Rubocop offences/violations caught as soon as this is merged, and so the CI will fail. Those need to be fixed in a separate PR.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,6 +48,9 @@ group :development, :test do
 end
 
 group :development do
+  # GOV.UK interpretation of rubocop for linting Ruby
+  gem 'govuk-lint'
```
Comment (dankmitchell) [inline Gemfile]: I think you'll want this in your test group too other travis/vsts can't use it...

Comment (himal-mandalia) [reply Gemfile]: Good catch, should have been in the development and test group.

**Comment by heidar** [comment]
I'm done adding to this PR now, but seems like the pipeline is failing. This needs looking into.

https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/results?buildId=1748

**Comment by himal-mandalia** [comment]
@heidar I re-queued the build and it passed. Weird. This may be flickering, will have to watch out for it and resolve.

**Pull request merged**